docs/source/cuvs_bench/index.rst

```diff
@@ -357,6 +357,25 @@ The usage of `python -m cuvs_bench.split_groundtruth` is:
       --groundtruth GROUNDTRUTH
                             Path to billion-scale dataset groundtruth file (default: None)
 
+Running the MG versions of benchmarks
```

Comment:
Can you rename this to multi-GPU? We need to expect that most end-users aren't going to be famliar w/ our acronyms.